src/main/java/org/jabref/model/study/QueryEntry.java

```diff
@@ -0,0 +1,50 @@
+package org.jabref.model.study;
+
+public class QueryEntry {
```

Comment:
I'm a bit undecided about this, but what do you think about simply `Query`? Or `StudyQuery`? Entry is a bit confusing with `BibEntry` etc.

Reply:
Similarily to above, maybe one can reuse the ComplexQuery here (with jackson parser directly giving a complex query back)?